cmd/root.go

```diff
@@ -69,7 +69,8 @@ func Execute() {
 	signal.Notify(shutdown, syscall.SIGINT, syscall.SIGTERM, os.Interrupt)
 
 	if err := rootCmd.Execute(); err != nil {
-		rootCmd.PrintErr(err)
+		// err is already printed out by cobra's Execute
+		os.Exit(1)
```

Comment:
I have a weird request for this that will make more sense in the future. Instead of calling `os.Exit()`, can you refactor this function signature to `func Execute() error` and return the error from here?